tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2376,6 +2376,24 @@ describe('scalar card', () => {
           }),
         ]);
       }));
+
+      it('does not render fobs when no timeSelection is provided', fakeAsync(() => {
+        store.overrideSelector(getMetricsLinkedTimeSelection, {
+          start: {step: 20},
+          end: null,
+        });
+        const fixture = createComponent('card1');
+        const scalarCardFobController = fixture.debugElement.query(
+          By.directive(ScalarCardFobController)
+        ).componentInstance;
+        delete scalarCardFobController.timeSelection;
```

Comment:
Why do we need to set the time selection and delete it here?

Reply:
Without #6006 the card fob controller won't exist when time selection is not defined.

Reply:
I see. Could you mark a TODO here to update the test in #6006? I feel it weird to use delete after #6006

Reply:
Sure, I'll add a TODO and refactor the test alongside #6006